app/controllers/users_controller.rb

```diff
@@ -1,3 +1,31 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
```

Comment:
```suggestion
    @user = User.find params[:id]
```

Reply:
Phân biệt giúp anh find và find_by

Reply:
Lúc em sửa thành find_by thì em không thấy nó đỏ lòm Exception khi tìm theo users/id, em tìm hiểu thì find_by nó return value về nil nếu trong record k có giá trị phù hợp. Còn find thì nó trả thẳng về Exception: ActiveRecord Not Found. Em cũng đã update phần này ở Chapter8. Với lại những lỗi về convention thì cũng đã fix ở Chapter8 rồi ạ

Reply:
À vâng ạ phần chapter8, em làm tiếp find_by theo email ở phần đăng nhập cũng hiểu thêm chút về find_by rồi ạ. Thank anh nhìu

Reply:
chỗ này nếu find k tìm thấy thì raise exception dẹo bố trang rồi
còn chạy xuống dc line 4 nữa k?
(e test bằng cách sửa ID user là 1 số rất lớn sẽ thấy app vẹo)

Reply:
Dạ vâng, em test thử trang đỏ lòm luôn ạ